README.md

```diff
@@ -132,6 +132,50 @@ until the constructor for `config.Provider` can be fully satisfied.
 If resolution is not possible (for instance, if one of the required dependencies
 lacks a constructor and doesn't appear in the graph), `dig` returns an error.
 
+## Invoke
+
+The `Invoke` API executes the provided function and stores returned results in the dig container.
+
+`Invoke` looks through the dig graph and resolves all the constructor parameters for execution.
+At the end of method invokation, `Invoke` inserts return results except the `error` object, into the
```

Comment:
- invokation --> invocation
- return results or returned results?
- results except --> results, except

Also what does "end of method invocation" mean? Like when the method is fully invoked?